config/settings/base.py

```diff
@@ -74,6 +75,7 @@
     "allauth.socialaccount",
     "waffle",
     "django_prometheus",
+    "corsheaders",
```

Comment:
I suspect that by making the changes to the .bin URLs in experiment.html I suggest you won't need any of this CORS stuff. Can you remove all this cors stuff in this file after making those updates and see?